js_modules/dagster-ui/packages/ui-core/.storybook/main.js

```diff
@@ -16,7 +16,7 @@ const config = {
     getAbsolutePath('@storybook/addon-mdx-gfm'),
   ],
   framework: {
-    name: getAbsolutePath('@storybook/nextjs'),
+    name: '@storybook/react-webpack5',
```

Comment:
oh weird that we needed to change this, did you make sure nothing broke with storybook?

Reply:
Yeah, it looks fine. I think this might have been automated during the storybook migration, but since `ui-core` doesn't know anything about Next, it doesn't have the `next` dependency, so it's not quite the right framework option here.